README.md

```diff
@@ -19,7 +19,7 @@ is `pactum` package itself and some knowledge of python.
 ```python
 import pactum
 
-class MyAPI(API):
+class MyAPI(pactum.API):
```

Comment:
@lamenezes I implemented the same import just using the name pactum.API just to show that the library can be used with just one import.